examples/err_cmd/README.md

```diff
@@ -0,0 +1,147 @@
+        "pfs": {
+            "glob": "/*",
+            "repo": "input"
+        }
+    },
+    "transform": {
+        "cmd": [ "bash" ] ,
+        "stdin": [ "if", "[ -a /pfs/input/file1 ]", "then cp /pfs/input/* /pfs/out/", "exit 0",  "fi", "exit 1" ] ,
+        "err_cmd": [ "bash" ] ,
+        "err_stdin": [ "if", "[ -a /pfs/input/file2 ]",  "then", "exit 0", "fi", " exit 1" ]
+    }
+}
+```
+
+In the pipeline above, the code checks if the datum contains `file1`. If it
+does, then the code copies everything in `/pfs/input/` to the `/pfs/out`
+directory. If the datum does not include `file1`, the datum is checked
+against the code in `err_stdin`. That code checks if the datum has
+`file2`. If it does, the code marks the datum as recovered, and the
+job succeeds. If it does not, the job fails.
+
+Create a pipeline by running the following command from the `examples/err_cmd/`
+directory:
+
+```bash
+$ pachctl create pipeline -f error_test.json
+```
+
+Verify that the pipeline was successfully created:
+
+```bash
+$ pachctl list pipeline
+NAME       VERSION INPUT    CREATED       STATE / LAST JOB
+error_test 1       input:/* 5 seconds ago running / starting
+```
+
+## Add Files to the Input Repository
+
+Now, let's add some files to the input repository to watch how your pipeline
+code and error code work.
+
+You will add three files, `file1`, `file2`, and `file3`, that each contains one
+line in them.
+
+1. Add `file1`:
+
+   ```bash
+   $ echo "foo" | pachctl put file input@master:file1
+   ```
+
+   When you add `file1`, your pipeline should succeed:
+
+   ```bash
+   $ pachctl list job --no-pager
+   ID                               PIPELINE   STARTED        DURATION           RESTART PROGRESS  DL UL STATE
+   c8860dae5a054ec38a33068f75fe9690 error_test 13 seconds ago Less than a second 0       1 + 0 / 1 4B 4B success
+   ```
+
+   As you can see in the `PROGRESS` column – `1 + 0 / 1`, you have one
+   successfully processed datum.
+
+1. Add `file2`:
+
+   ```bash
+   $ echo "bar" | pachctl put file input@master:file2
+   ```
+
+   Processing of this datum will succeed as well, but the datum will be marked
+   as *recovered* because it does not contain `file1` as per our pipeline code:
+
+   ```bash
+   $ pachctl list job --no-pager
+   ID                               PIPELINE   STARTED       DURATION           RESTART PROGRESS      DL UL STATE
+   bc3da288ff884d5a9bcb312dd6cf07cb error_test 3 seconds ago Less than a second 0       0 + 1 + 1 / 2 0B 0B success
+   c8860dae5a054ec38a33068f75fe9690 error_test 3 minutes ago Less than a second 0       1 + 0 / 1     4B 4B success
+   ```
+
+   In the `PROGRESS` column, you can see that the last job did not have
+   any successfully processed datums, but it had a skipped datum and a
+   recovered datum – `0 + 1 + 1 / 2`.
+
+1. Add `file3`:
+
+
+   ```bash
+   $ echo "baz" | pachctl put file input@master:file3
+   ```
+
+   Because the processed datum does not have nor `file1`, neither
```

Comment:
incorrect usage of `nor`

Reply:
Because the processed datum has neither `file1 ` nor `file2, the third datum fails and therefore the entire job fails.